nexus/tests/integration_tests/basic.rs

```diff
@@ -476,10 +476,10 @@ async fn test_projects_list(cptestctx: &ControlPlaneTestContext) {
     for _ in 0..projects_total {
         // We'll use uuids for the names to make sure that works, and that we
         // can paginate through by _name_ even though the names happen to be
-        // uuids.  Names have to start with a letter, though, so we've got to
-        // make sure our uuid has one.
+        // uuids.  Names have to start with a letter and can't exactly match
```

Comment:
I can't believe we actually had a case doing exactly this!  How was it working before?

Reply:
Do you mean the test? The first character of the UUID was replaced with a lowercase ascii character so it was a bit artificial.

Reply:
Oh of course 🤦